aemedge/blocks/video/video.js

```diff
@@ -87,14 +108,18 @@ const loadVideoEmbed = (block, link, replacePlaceholder, autoplay) => {
   const isYoutube = link.includes('youtube') || link.includes('youtu.be');
   const isVimeo = link.includes('vimeo');
   const isMp4 = link.includes('.mp4');
-
+  const isSling = link.includes('.sling');
   let embedEl;
   if (isYoutube) {
+    console.log('yt called');
     embedEl = embedYoutube(url, replacePlaceholder, autoplay);
   } else if (isVimeo) {
     embedEl = embedVimeo(url, replacePlaceholder, autoplay);
   } else if (isMp4) {
     embedEl = getVideoElement(link, replacePlaceholder, autoplay);
+  } else if (isSling) {
+    console.log('sling called');
```

Comment:
this console log is fine, but can you log it as 'watch.sling is called' just to be more specific?